src/main/java/com/ericsson/ei/controller/RuleControllerImpl.java

```diff
@@ -34,49 +31,55 @@
 import com.ericsson.ei.controller.model.RuleCheckBody;
 import com.ericsson.ei.controller.model.RulesCheckBody;
 import com.ericsson.ei.jmespath.JmesPathInterface;
+import com.ericsson.ei.rules.RulesHandler;
 import com.ericsson.ei.services.IRuleCheckService;
 import com.ericsson.ei.utils.ResponseMessage;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
 
 import io.swagger.annotations.Api;
 import io.swagger.annotations.ApiOperation;
 import io.swagger.annotations.ApiParam;
 import lombok.Setter;
 
-/**
- * This class implements the Interface for JMESPath API, generated by RAML 0.8
- * Provides interaction with JmesPathInterface class
- *
- * Usage: 1. If input contains a rule as an argument and json expression in a
- * text file, then the following curl command may be used. curl -H
- * "Content-type: application/x-www-form-urlencoded" -X POST --data-urlencode
- * jsonContent@testjson.txt
- * http://localhost:8090/jmespathrule/runRule?rule=data.outcome
- *
- * 2. If input contains rule and json expression as two String arguments, then
- * the following curl command may be used. curl -H "Content-type:
- * application/x-www-form-urlencoded" -X POST -d
- * jsonContent={"data":{"outcome":{"conclusion":"SUCCESSFUL"},"test":"persistentLogs"}}
- * http://localhost:8090/jmespathrule/runRule?rule=data.outcome
- *
- */
-
 @Component
 @CrossOrigin
-@Api(value = "checkRules", tags = {"Check rules"})
-public class RuleCheckControllerImpl implements RuleCheckController {
+@Api(value = "Rules", tags = {"Rules"})
+public class RuleControllerImpl implements RuleController{
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(RuleCheckControllerImpl.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(RuleControllerImpl.class);
 
     @Autowired
     private JmesPathInterface jmesPathInterface;
 
     @Autowired
     private IRuleCheckService ruleCheckService;
 
+    @Autowired
+    private RulesHandler rulesHandler;
+
     @Setter
     @Value("${testaggregated.enabled:false}")
     private Boolean testEnable;
 
+    @Override
+    @CrossOrigin
+    @ApiOperation(value = "Get the active rules file content", response = String.class)
```

Comment:
Update the ApiOperation here, since it's possible to use rules from URL and files. Maybe keep it as "Get the active rules from Eiffel Intelligence" :)

Reply:
Done.